hyco-https/lib/_hyco_outgoing.js

```diff
@@ -780,7 +784,9 @@ OutgoingMessage.prototype._flushOutput = function _flushOutput(socket) {
   var output = this.output;
   var outputCallbacks = this.outputCallbacks;
   for (var i = 0; i < outputLength; i++) {
-    ret = socket.send(output[i], { binary: this.outputFrameBinary[i], fin : false}, outputCallbacks[i]);
+    // Response sent through the control connection must have its own frame
+    var isHeaderThroughControlConnection = this._controlChannel && i === 0;
```

Comment:
>isHeaderThroughControlConnection [](start = 8, length = 32)

This makes it "appear" (not saying this actually happens) that the {response} JSON object through the data/rendezvous connection doesn't need fin: true which seems odd.